### PR TITLE
assessment complete

### PR DESCRIPTION
**Pull request by TheLavitz**
# Submitting Week 5 Assessment

### ✅ Completed Files

Select the following as confirmation that the files in this repository are complete.

- [X] Career Readiness
- [X] Coding with AI
- [X] Interview Questions
- [X] JavaScript Challenges
- [X] Ruby Challenges

### ✋ Reminder

Do NOT merge your assessment!

### 📝 Feedback

If you would like specific feedback please leave details here:

### CODE REVIEW

File: coding-with-ai.rb [hunk 1]
```diff
@@ -6,11 +6,16 @@
...
-end
+  result[vote.downcase] += 1 #the result hash is then accessed and string held by the vote parameter is located among result's key and its value is incremented by 1
+  result #we then return the result hash in its entirety
+end #and end the do block
 
-p totals
\ No newline at end of file
+p totals #the totals variable is then printed which returns the result hash in its entirety as it is the last line of the .reduce method being called
+
+#"You have received a list of users' favorite tacos. Using Ruby, write the code that will create a hash containing all of the different tacos listed and the number of votes each kind received."
```
Comment (elysemontano) [inline coding-with-ai.rb:21]: Excellent work!

File: javascript-challenges.test.js [hunk 2]
```diff
@@ -14,30 +14,98 @@
...
 
 // b) Create the function that makes the test pass.
 
 // Pseudo code:
+// Declare a function called encoder
+// Input: string
+// Output: string
+// Declare a variable called encoded to hold the output of the function
+// Use .replace() to to exchange each letter with the number it corresponds with
+// Use regular expressions /a/gi in place of "a" to look for all of the "a" in the string as indicated by the g in /a/gi and to perform the replacement on upper and lowercase versions of "a" as indicated by the i in /a/i
```
Comment (elysemontano) [inline javascript-challenges.test.js:43]: Awesome pseudocode and explanation!

File: javascript-challenges.test.js [hunk 3]
```diff
@@ -14,30 +14,98 @@
...
+// Input: Array of numbers
+// Output: Boolean
+// Declare an object called counter to hold the count of each number in the array
+// Use a for loop to step through the array and check counter for the value at that index of the array and increment the count by 1 for every instance of the value it come accross
+// Use another for loop to step through the array and check if the value at that index in the array has a count in counter that is either 2 or 3.
+// If an array has a value with a count of 2 let a variable pair equal true
+// If an array has a value with a count of 3 let a variable threeOfAKind equal true
+// If pair and threeOfAKind both equal true then the function will return true
+
+const full_house = (array) => {
```
Comment (elysemontano) [inline javascript-challenges.test.js:93]: Logic looks great in this section!  If you want to stretch yourself, consider some edge cases in your tests and refactoring your code.

**Review by elysemontano — APPROVED** [review]
Once again, an outstanding assessment displaying a competent skill level!  I left a comment in coding challenges about stretching yourself by considering edge cases in your test and refactoring your code, but overall this assessment looks great!
